src/test/java/GS/test/GSBufferTests.java

```diff
@@ -88,13 +88,7 @@ void GSBuffer_VoltageToIntConversion_limit_low()
             buffertest = new GSBuffer( 2000, 64);
         } catch (Exception ex) {fail(ex);}
 
-        try {
-            buffertest.appendValue(-10,1);
-        } catch(ActiveChanException ACex) {
-            fail(ACex);
-        } catch(VoltageRangeException VRex){
-            System.out.println("voltage limit low pass");
-        }
+        assertThrows(VoltageRangeException.class, () -> buffertest.appendValue(-10,1));
     }
```

Comment:
can you add the "voltage limit low pass" statement here?

Reply:
done